homeassistant/components/hunterdouglas_powerview/config_flow.py

```diff
@@ -44,13 +44,9 @@ async def validate_input(hass: core.HomeAssistant, hub_address: str) -> dict[str
         raise CannotConnect from err
 
     if hub.role != "Primary":
-        _LOGGER.debug(
-            "%s (%s) is the %s Hub. Only the Primary can manage shades",
-            hub.name,
-            hub.hub_address,
-            hub.role,
+        raise UnsupportedDevice(
+            f"{hub.name} ({hub.hub_address}) is the {hub.role} Hub. Only the Primary can manage shades"
```

Comment:
Please break long strings around max 88 characters per line.

Reply:
This line is actually exactly 88 characters long already. (Unless you are including leading spaces?)

Importantly for my context too though, is this to do with formatting for HA backend or the user interface?

Reply:
> This line is actually exactly 88 characters long already. (Unless you are including leading spaces?)
> 

Actual line length so it doesn't exceed the screen width on small screens

> Importantly for my context too though, is this to do with formatting for HA backend or the user interface?

Its for code readability